activestorage/test/models/attached/one_test.rb

```diff
@@ -806,6 +806,21 @@ def self.name; superclass.name; end
     assert_equal 67, image.height
   end
 
+  test "creating variation with dynamic variation" do
+    john = User.create!(name: "John")
+
+    assert_no_enqueued_jobs only: ActiveStorage::TransformJob do
+      @user.avatar_with_dynamic.attach fixture_file_upload("racecar.jpg")
+      john.avatar_with_dynamic.attach fixture_file_upload("racecar.jpg")
+    end
+
+    josh_variantion = @user.avatar_with_dynamic.variant(:thumb).variation
+    john_variantion = john.avatar_with_dynamic.variant(:thumb).variation
+
+    assert_equal({ format: "jpg", resize_to_limit: [100, 100] }, josh_variantion.transformations)
+    assert_equal({ format: "jpg", resize_to_limit: [200, 200] }, john_variantion.transformations)
```

Comment:
typo fix: s/variantion/variation

```suggestion
    josh_variation = @user.avatar_with_dynamic.variant(:thumb).variation
    john_variation = john.avatar_with_dynamic.variant(:thumb).variation

    assert_equal({ format: "jpg", resize_to_limit: [100, 100] }, josh_variation.transformations)
    assert_equal({ format: "jpg", resize_to_limit: [200, 200] }, john_variation.transformations)
```